debezium-api-contracts/src/main/resources/mongodb/mongodb-1.5.0.Final.json

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
So is this here as an example only, as this would be created on the fly by the generator eventually?

Reply:
Yep. Example only. We need to clarify when and how this is generated and deployed.